h/stcbase.h

```diff
@@ -78,6 +78,7 @@ typedef struct STCModule_tag {
   int  (*udpHandler)(struct STCBase_tag *stcBase, struct STCModule_tag *module, Socket *socket);
   int  (*workElementHandler)(struct STCBase_tag *stcBase, struct STCModule_tag *stcModule, WorkElementPrefix *prefix);
   int  (*backgroundHandler)(struct STCBase_tag *stcBase, struct STCModule_tag *stcModule, int selectReturnCode);
+  char *sem_table_pointer;
```

Comment:
You're putting something very specific into a very generic header. This cannot be here.

Reply:
Yes, that was a tryout, which I have removed everywhere, but I missed that one.  I will remove it now.